components/Voorbeelden/checken.config.yml

```diff
@@ -28,37 +28,35 @@ context:
     - title: Plaats van het bouwwerk
       id: panel3
       open: true
-      form:
-        fieldsets:
-          - legend: Plaats van het bouwwerk
-            groups:
-              - id: omgevingsvergunningPostcode
-                inputType: input
-                type: text
-                label: Postcode
-                size: 8
-              - id: omgevingsvergunningHuisnummer
-                inputType: input
-                type: text
-                label: Huisnummer
-                size: 8
-              - id: omgevingsvergunningToevoeging
-                inputType: input
-                type: text
-                label: Huisnummer toevoeging
-                size: 8
-              - id: omgevingsvergunningStraatnaam
-                inputType: input
-                type: text
-                label: Straatnaam
-              - id: omgevingsvergunningWoonplaats
-                inputType: input
-                type: text
-                label: Woonplaats
-        buttons:
-        - type: submit
-          modifier: primary
-          label: Volgende stap
+      fieldsets:
```

Comment:
Je hebt nu hier `form` weggehaald, @PieterVerheij, maar die moet dan nog wel elders in de pagina eromheen (in `checken.njk` dus, denk ik). Zoals je bij `aanvragen.njk` wel hebt gedaan. Of, wil je hiermee laten zien dat het voor de styling an sich niet nodig is...?

Reply:
Zelfs als je wat 'wil laten zien' lijkt dit me niet het voorbeeld. Dit voorbeeld moet gewoon goed zijn: er wordt user input verwacht dus een `<form>` om de hele werkvorm, zoals elders, is noodzakelijk.

Reply:
@bramhoosemans , of dat dan een `form-horizontal` moet zijn (zoals bv. in `aanvragen-file-upload.njk`) is mij niet helemaal duidelijk. Ik denk dat in iedere geval in alle voorbeelden van 1 flow die form-typering eenduidig moet zijn, in eerdere discussies zijn we er ook op terechtgekomen dat deze vormfactor niet per stap zal verschillen.